src/hooks/useApproveConfirmTransaction.ts

```diff
@@ -129,6 +131,7 @@ const useApproveConfirmTransaction = ({
         })
         .on('receipt', (payload: Web3Payload) => {
           dispatch({ type: 'approve_receipt', payload })
+          onApproveSuccess(state)
```

Comment:
Note sure if `state` would ever be useful within the function tbh, I just mirrored the existing `onSuccess` func.